example/over_react_redux/components/counter.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:developer';
+import 'dart:html';
+
+import 'package:over_react/over_react.dart';
+import 'package:over_react/over_react_redux.dart';
+import '../store.dart';
+
+part 'counter.over_react.g.dart';
+
+@Factory()
+UiFactory<CounterProps> Counter = _$Counter;
+
+@Props()
+class _$CounterProps extends UiProps with ConnectPropsMixin {
+  int intProp;
+
+  Map<String, dynamic> customStyles;
+
+  Function dispatcherProp;
+}
+
+@Component2()
+class CounterComponent extends UiComponent2<CounterProps> {
+  @override
+  render() {
+    return (Dom.div()..style = props.customStyles)(
+        Dom.div()('Count: ${props.intProp}'),
+        (Dom.button()..onClick = (_){ props.dispatch != null ? props.dispatch(new IncrementAction()) : '';})('+1'),
```

Comment:
I don't know if this component is the best place, but would it be beneficial to show an example with `mapDispatchToProps`?